amundsen_common/models/lineage.py

```diff
@@ -0,0 +1,34 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Optional, List
+
+import attr
+from marshmallow_annotations.ext.attrs import AttrsSchema
+
+
+@attr.s(auto_attribs=True, kw_only=True)
+class LineageItem:
```

Comment:
could you share some examples on this class? e.g what are the badges and usage using here?

Reply:
Just added the actual Badge class for badge list. Not sure what you mean by what the usage is using here?

Reply:
we can change `usage` to be even more general and make it something like `sorting_metric` or something like that, but usage seems like a natural way in which we would want to highlight more important upstream/downstream resources when you have lots of them

Reply:
My question is around how the metadata lineage entity connection will look like with badge and usage.